docs/modules/plugins/partials/tutorial-web-steps.adoc

```diff
@@ -0,0 +1,340 @@
+|By.linkUrl(http://httpbin.org)
+
+|CLICK_AND_HOLD
+|Element locator or empty
+|By.linkText(Click me)
+
+|MOVE_BY_OFFSET
+|Point
+|(10, 15) where *x* is 10 and *y* is 15
+
+|RELEASE
+|Element locator or empty
+|By.tagName(div)
+
+|ENTER_TEXT
+|Text to type
+|Minsk City
+
+|CLICK
+|Element locator or empty
+|By.caseSensitiveText(Done)
+
+|PRESS_KEYS
+|Press and release any of https://selenium.dev/selenium/docs/api/java/org/openqa/selenium/Keys.html[Keys]
+|BACK_SPACE
+
+|KEY_DOWN
+|Press any of https://selenium.dev/selenium/docs/api/java/org/openqa/selenium/Keys.html[Keys] one by one
+|CONTROL,SHIFT,ALT
+
+|KEY_UP
+|Release any of https://selenium.dev/selenium/docs/api/java/org/openqa/selenium/Keys.html[Keys] one by one
+|CONTROL,SHIFT,ALT
+
+|MOVE_TO
+|Element locator
+|By.id(username)
+
+|===
+
+.Execute various web-actions
+[source,gherkin]
+----
+When I execute sequence of actions:
+|type          |argument                                |
+|DOUBLE_CLICK  |By.fieldText(Hello World)               |
+|DOUBLE_CLICK  |                                        |
+|CLICK_AND_HOLD|By.xpath(//signature-pad-control/canvas)|
+|CLICK_AND_HOLD|                                        |
+|MOVE_BY_OFFSET|(-300, 0)                               |
+|RELEASE       |By.xpath(//signature-pad-control/canvas)|
+|RELEASE       |                                        |
+|ENTER_TEXT    |Text                                    |
+|CLICK         |By.placeholder(Enter your password)     |
+|CLICK         |                                        |
+|PRESS_KEYS    |BACK_SPACE                              |
+|KEY_DOWN      |CONTROL,SHIFT                           |
+|KEY_UP        |CONTROL,SHIFT                           |
+|MOVE_TO       |By.id(name)                             |
+----
+
+[TIP]
+====
+This step can be used to perform clipboard interactions.
+
+.Select all text in the focused field and copy it to the clipboard on Windows
+[source,gherkin]
+----
+When I execute sequence of actions:
+|type      |argument  |
+|KEY_DOWN  |CONTROL, a|
+|KEY_UP    |a, CONTROL|
+|KEY_DOWN  |CONTROL, c|
+|KEY_UP    |c, CONTROL|
+----
+
+.Paste text from the clipboard to the focused field on MacOS
+[source,gherkin]
+----
+When I execute sequence of actions:
+|type      |argument  |
+|KEY_DOWN  |COMMAND, v|
+|KEY_UP    |v, COMMAND|
+----
+====
+
+=== Browser logs steps
+
+This set of steps allows to validate the https://developer.mozilla.org/en-US/docs/Web/API/console[browser console logging messages].
+
+:log-levels: List of the comma-separated messages levels. The supported levels are: ERRORS, WARNINGS, INFOS.
+
+[TIP]
+=====
+In order to configure availability of the INFO level messages use following properties:
+[cols="1,2"]
+|===
+
+|Browser
+|Property to enable INFO logs
+
+|Google Chrome
+|`selenium.capabilities.goog\:loggingPrefs.browser=INFO`
+
+|Microsoft Edge Chromium
+|`selenium.capabilities.ms\:loggingPrefs.browser=INFO`
+|===
+=====
+
+==== Validate log entries absence
```

Comment:
is it really crucial part to be added to the tutorial?

Reply:
I wouldn't say it's crucial, but detaching just specific steps will complicate overall structure and maintenance in my opinion. This tutorial isn't considered as the first step in learning course and thus might contain some "specific" info.